main.js

```diff
@@ -0,0 +1,133 @@
+let currentBoard = { 
+  }
+
+function newGame(){
+  currentBoard[0]= [5,4,3,2,1],
+  currentBoard[1]= [];
+  currentBoard[2]=[];
+  return(currentBoard)
+}
+function showBoard(){
+  console.log(Object.entries(currentBoard).map((i) => {return("---" + i[1].join(" "))}))
+}
+
+newGame()
+
+function moveDisc(disc, peg){
+  // let lastPeg = peg[peg.length-1];
+  let lastDiscPeg1 = currentBoard[0][currentBoard[0].length-1];
+  let lastDiscPeg2 = currentBoard[1][currentBoard[1].length-1];
+  let lastDiscPeg3 = currentBoard[2][currentBoard[2].length-1];
```

Comment:
DRY, dont repeat yourself
you could have defined a function that takes an index and then you pass the index. 
```
const getPeg = (index) => currentBoard[index][currentBoard[index].length-1];
```
then you do 
```
  let lastDiscPeg1 = getPeg(1);
```